ui/gtk3/emojier.vala

```diff
@@ -1242,6 +1242,44 @@ public class IBusEmojier : Gtk.ApplicationWindow {
         unowned GLib.SList<string>? sub_emojis = null;
         unowned GLib.SList<unichar>? sub_exact_unicodes = null;
         unowned GLib.SList<unichar>? sub_unicodes = null;
+        // Create Pango context for font checking
+        var label = new Gtk.Label("");
+        var pango_context = label.get_pango_context();
+        bool is_emoji_supported(string emoji) {
+            var layout = new Pango.Layout(pango_context);
+            var font_desc = Pango.FontDescription.from_string("Noto Color Emoji");
```

Comment:
What happens if "Noto Color Emoji" is not the default emoji font? Is it OK to hardcode "Noto Color Emoji" here? 

Maybe use 'emoji' instead?:

mfabian@f41:~$ fc-match emoji
NotoColorEmoji.ttf: "Noto Color Emoji" "Regular"
mfabian@f41:~$

Reply:
`IBusEmojier.EWhiteLabel` includes the custom font `m_emoji_font_family`.